pkg/helm/actions/install_chart.go

```diff
@@ -5,38 +5,50 @@ import (
 	"helm.sh/helm/v3/pkg/chart"
 	"helm.sh/helm/v3/pkg/chart/loader"
 	"helm.sh/helm/v3/pkg/release"
+	"k8s.io/client-go/util/retry"
 )
 
 func InstallChart(ns, name, url string, vals map[string]interface{}, conf *action.Configuration) (*release.Release, error) {
-	cmd := action.NewInstall(conf)
+	var release *release.Release
 
-	releaseName, chartName, err := cmd.NameAndChart([]string{name, url})
-	if err != nil {
-		return nil, err
-	}
-	cmd.ReleaseName = releaseName
+	err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		cmd := action.NewInstall(conf)
```

Comment:
Can we just retry around the run call instead of the prep calls?

Reply:
Originally was concerning the setup step plays a role in `"cannot re-use a name that is still in use"` error. Just tested wrapping around only `cmd.Run` and it worked. Will update.

Reply:
Updated.